src/mbgl/shader/raster_shader.hpp

```diff
@@ -5,11 +5,18 @@
 
 namespace mbgl {
 
+namespace gl {
+template <class> class VertexBuffer;
+} // namespace gl
+
+class RasterVertex;
+
 class RasterShader : public Shader {
 public:
     RasterShader(gl::Context&, Defines defines = None);
 
-    void bind(int8_t* offset) final;
+    void bind(const gl::VertexBuffer<RasterVertex>&,
+              const int8_t* offset);
```

Comment:
My plan here is to replace both parameters with `const gl::VertexBufferSlice<RasterVertex>&`, but I'd like to put that off until a followup PR.